content/blog/merging-local-and-remote-apt-repositories.md

```diff
@@ -0,0 +1,200 @@
++++
+title = 'Merging Local and Remote Apt Repositories'
+date = 2018-11-29T11:28:13-08:00
+draft = true
+tags = ["tags"]
+description = "Desc"
+
+# For twitter cards, see https://github.com/mtn/cocoa-eh-hugo-theme/wiki/Twitter-cards
+meta_img = "/images/image.jpg"
+
+# For hacker news and lobsters builtin links, see github.com/mtn/cocoa-eh-hugo-theme/wiki/Social-Links
+hacker_news_id = ""
+lobsters_id = ""
++++
+
+> sudo apt-get install -y docker-ce
+
+If you're using `docker-ce` right now chances are that you proably installed it
+with the above command, but the maintenance of the repositories that hold that
+artifact can prove troublesome due to the time and environment in which the
+repository maintenance tooling was created.
+
+Entire industries have been created over the idea that debian repositories are
+difficult to maintain and a career field (of which I also work) has spawned due
```

Comment:
`s/of which/in which/g` or remove the brackets and add after the sentence as an aside.
E.g `... this type of work can be - I know because I work in this field.`